src/webview/view/components/forms/spfxProject/FormHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export interface IFormHeaderProps {
```

Comment:
Let's keep two break lines betewwn the imports and the rest of the code

```suggestion


export interface IFormHeaderProps {
```